GNSS/src/GNSSManager.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef GNSS_MANAGER_H
+#define GNSS_MANAGER_H
+
+#include <Arduino.h>
+#include "rover_lib2/helpers/log.hpp"
+#include "rover_lib2/helpers/macros.hpp"
+#include "rover_lib2/sensors/gnss_parser.hpp"
+#include "rover_lib2/filters/circular_moving_average.hpp"
+
+struct sGNSSData
+{
+    float latitude = 0.0f;
```

Comment:
[nitpick]
use capitalized user literral
```suggestion
float latitude = 0.0F;
```

apply everywhere